app/controllers/interaction_claims_controller.rb

```diff
@@ -55,4 +51,40 @@ def perform_interaction_search
     search_results = LookupInteractions.find(params)
     @search_results = InteractionSearchResultsPresenter.new(search_results, view_context)
   end
+
+  def unpack_locals params
+    @preset_fda = (params[:fda_approved_drug] == "checked" ? "FDA Approved" : "")
+    @preset_neo = (params[:anti_neoplastic] == "checked" ? "Anti-neoplastics" : "")
+    @preset_immuno = (params[:immunotherapy] == "checked" ? "Immunotherapies" : "")
+    @preset_clin = (params[:clinically_actionable] == "checked" ? "Clinically Actionable" : "")
+    @preset_druggable = (params[:druggable_genome] == "checked" ? "Druggable Genome" : "")
+    @preset_resist = (params[:drug_resistance] == "checked" ? "Drug Resistance" : "")
+  end
+
+  def prepare_export(compound_field_separator = '|')
+    headers = %w[search_term match_term match_type gene drug interaction_types sources pmids]
+    @tsv = CSV.generate(col_sep: "\t") do |tsv|
+      tsv << headers
+      @search_results.instance_variable_get(:@search_results).each do |result|
```

Comment:
Rather than using reflection here with `instance_variable_get` you could always just add an accessor method in the search results class `attr_reader :search_results` and then call `@search_results.search_results` directly. Likewise for `type`, `search_term`, etc